plugins/GSdx/res/glsl/tfx_fs.glsl

```diff
@@ -842,6 +855,11 @@ void ps_main()
     return;
 #endif
 
+#if !SW_BLEND && PS_DITHER
+    ps_dither(C);
+    C.rgb -= 3.0;
```

Comment:
What does the `-= 3.0` do?

Reply:
The dither matrix values range is [-4, 3], so that -3 makes the range [-7, 0]. This is to avoid incrementing values as it causes artifacts to show up (see screenshots in comments). There's no such problem when software blending is used.

Reply:
It'd be worth adding a comment to the code explaining this (it can wait till after the other PR is merged though).

Reply:
Done.